### ISSUE TITLE
How to interpret results like "Segments=0+,12+,0-"?

### ISSUE THREAD

**Issue by KR-Chow** [jluebeck/PrepareAA#6]
Hi Jens,

How to interpret results like "Segments=0+,12+,0-"? There is only one segment in this cycle.

Best,
Keren

**Comment by jluebeck** [comment]
Hi Keren,

This indicates a single segment, which is joined to the expected endpoints
for the segment in the reference genome.

Jens

**Comment by KR-Chow** [comment]
Hi Jens,

Thanks for your reply. What puzzles me is about the expected endpoints. Could you please explain more about it?

Best,
Keren

**Comment by jluebeck** [comment]
If I understand the question correctly, in this case AA identified a segment of the genome (which was presumably amplified, though I cannot tell without seeing the estimated CN). The endpoints of the segment are determined by the segmentation analysis performed by AA. No discordant edges passing the AA filtering criteria were identified connecting the endpoints of the (presumably) amplified region to other, non-native locations in the genome.

**Comment by KR-Chow** [comment]
> If I understand the question correctly, in this case AA identified a segment of the genome (which was presumably amplified, though I cannot tell without seeing the estimated CN). The endpoints of the segment are determined by the segmentation analysis performed by AA. No discordant edges passing the AA filtering criteria were identified connecting the endpoints of the (presumably) amplified region to other, non-native locations in the genome.
 
So according to your explanation, cases like this can be ignored when identifying ecDNAs?

Best,
Keren

**Comment by jluebeck** [comment]
Correct, it does not have the discordant edges required to produce a cycle.

**Comment by KR-Chow** [comment]
Thank you so much, Jens! 

Stay safe during this challenging time!

Best,
Keren
